### ISSUE TITLE
Dynamic requires are not currently supported by rollup-plugin-commonjs

### ISSUE THREAD

**Issue by fluentart** [Hargne/html-creator#1]
Hi, I'm trying to use html-creator in typescript and during compilation I get the error:
```
/node_modules/html-creator/dist/main.js:11
	throw new Error('Dynamic requires are not currently supported by rollup-plugin-commonjs');
```
Any idea before I dig through the code?

**Comment by Hargne** [comment]
Hi,

Thanks for reporting this.
It is this line 9 in _src/tools.js_:
`const packageJson = require(path.join(process.cwd(), 'package.json'));`

As stated in the error, Rollup does not dynamic requires, and it is a little bit weird that this error has appeared sooner (because it should have :D). I'll look into this ASAP!

**Comment by Hargne** [comment]
This is now fixed in v0.4.3
